_data/simple-icons.json

```
@@ -478,7 +478,7 @@
         {
             "title": "Apache NetBeans IDE",
             "hex": "1B6AC6",
-            "source": "https://netbeans.apache.org/images/"
+            "source": "https://apache.org/logos"
```

Could you add here a foward slash at the end to maintain the consistency with other sources?

```suggestion
            "source": "https://apache.org/logos/"
```